chart/Chart.yaml

```diff
@@ -3,5 +3,5 @@ name: py-kube-downscaler
 description: A Helm chart for deploying py-kube-downscaler
 
 type: application
-version: 0.2.0
-appVersion: "24.5.0"
+version: 0.2.1
+appVersion: "24.5.1"
```

Comment:
is a new app release planned? Otherwise it's enough to increase the Helm chart version.